tests/CMakeLists.txt

```
@@ -6,16 +10,24 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(Catch2)
 
-add_executable(cpprealm_sync_tests
-                main.hpp
-                main.cpp
-                admin_utils.hpp
-                admin_utils.cpp
-                sync/test_objects.hpp
-                sync/asymmetric_object_tests.cpp
-                sync/flexible_sync_tests.cpp
-                sync/app_tests.cpp
-                sync/client_reset_tests.cpp)
+if (DEFINED USES_CONAN OR DEFINED VCPKG_OVERLAY_PORTS)
```

```suggestion
if (DEFINED USES_CONAN OR DEFINED VCPKG_TOOLCHAIN)
```
`VCPKG_TOOLCHAIN` is always set by the vcpkg toolchain file, whereas `VCPKG_OVERLAY_PORTS` might not be defined.